src/metadata_export.py

```diff
@@ -353,6 +354,26 @@ def save_release(self, data_record, node_type, node_id, crdc_id):
                 if result and children and len(children) > 0: 
                     self.delete_release_children(children)
     
+    def combine_parents(self, node_type, release_parents, node_parents):
+        if not release_parents or len(release_parents) == 0:
+            return node_parents
+        if not node_parents or len(node_parents) == 0:
+            return release_parents
+        relationships = self.model.get_node_relationships(node_type)
+        if node_parents and len(node_parents):
+            for parent in node_parents:
+                relationship = relationships[parent["parentType"]]["dest_node"]
+                if not dict_exists_in_list(release_parents, parent, keys=["parentType", "parentIDPropName", "parentIDValue"]):
+                    if relationship == "many_to_many":
+                        release_parents.append(parent)
```

Comment:
Please make sure there will be only one copy of the same parent, if new data contains same parent(s) as previous ones.

Reply:
If new data contains the same parent, the existing release record will not be changed.